views/play.erb

```diff
@@ -0,0 +1,9 @@
+
+<form method="get" action="/play">
+  <h1>Choose your first move:</h1>
+    <form method="get" action="/play">
+    <input type="submit" value="Rock">
+    <input type="submit" value="Paper">
+    <input type="submit" value="Scissors">
+</html>
+
```

Comment:
Question: How would you know which action the player made, if the post is always going to the same endpoint (nothing wrong here, just wondering how you might do it)